src/store/modules/bitcoin.js

```diff
@@ -118,6 +118,7 @@ const mutations = {
     const mergedBlocks = [...blocks, ...state.blocks];
     // remove duplicate blocks
     const uniqueBlocks = mergedBlocks.filter((v, i, a) => a.findIndex(t => (t.height === v.height)) === i);
+    
```

Comment:
Please remove this leftover newline.

Reply:
personal preference to have more space in my code but sure

Reply:
I'm no member of the Umbrel team, but I think code style changes in other files shouldn't be part of another PR, but the Umbrel team might see thing differently